src/test/java/step1/InputViewTest.java

```diff
@@ -0,0 +1,33 @@
+package step1;
+
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatIllegalArgumentException;
+
+
+public class InputViewTest {
+
+    @Test
+    void split_테스트() {
+        String input = "1 + 2 * 3";
+        InputView inputView = new InputView();
+        assertThat(inputView.splitInput(input)).containsExactly("1", "+", "2", "*", "3");
+    }
+
+    @Test
+    void 공백_검증() {
+        InputView inputView = new InputView();
+        assertThatIllegalArgumentException().isThrownBy(() -> {
+            inputView.emptyValidate("");
+        });
```

Comment:
도메인에서 IllegalArgumentException과 함께 오류 메세지도 던져주신 후 테스트로 메세지도 검증해보세요 :) 
```suggestion
        }).withMessage("에러메세지")
```